changelog/unreleased/enhancement-add-sse-event-for-locking-renaming-deleting-restoring

```diff
@@ -0,0 +1,12 @@
+Enhancement: Add SSE events for locking, renaming, deleting, and restoring
+
+We've added Server-Sent Events (SSE) for locking, renaming, deleting, and restoring actions.
+This enhancements ensures that these actions are instantly propagated to other browsers and devices where the user is logged in.
```

Comment:
```suggestion
This enhancement ensures that these actions are instantly propagated to other browsers and devices where the logged in user has access the relevant files and folders.
```
Not 100% happy about my change, but we should make clear that a logged in space member gets the events as well, even when a different space member triggers the action.